tests/basic.rs

```diff
@@ -912,69 +925,6 @@ fn test_addr_cmp_hash_eq() {
     assert!(!chan1.same_actor(&chan2));
 }
 
-#[tokio::test]
-async fn stop_in_started_actor_stops_immediately() {
-    let (_address, ctx) = Context::new(None);
-    let fut = ctx.run(StopInStarted);
-
-    fut.now_or_never().unwrap(); // if it stops immediately, this returns `Some`
-}
-
-struct StopInStarted;
-
-#[async_trait]
-impl Actor for StopInStarted {
-    type Stop = ();
-
-    async fn started(&mut self, ctx: &mut Context<Self>) {
-        ctx.stop_self();
-    }
-
-    async fn stopped(self) -> Self::Stop {}
-}
-
-#[tokio::test]
-async fn stop_all_stops_immediately() {
-    let (_address, ctx) = Context::new(None);
-
-    let fut1 = ctx.clone().run(InstantShutdownAll {
-        stop_all: true,
-        number: 1,
-    });
-    let fut2 = ctx.clone().run(InstantShutdownAll {
-        stop_all: false,
-        number: 2,
-    });
-    let fut3 = ctx.run(InstantShutdownAll {
-        stop_all: false,
-        number: 3,
-    });
-
-    fut1.now_or_never().expect("Should stop immediately");
-    fut2.now_or_never().expect("Should stop immediately");
-    fut3.now_or_never().expect("Should stop immediately");
-}
-
-struct InstantShutdownAll {
-    stop_all: bool,
-    number: u8,
-}
-
-#[async_trait]
-impl Actor for InstantShutdownAll {
-    type Stop = ();
-
-    async fn started(&mut self, ctx: &mut Context<Self>) {
-        if self.stop_all {
-            ctx.stop_all();
-        }
-    }
-
-    async fn stopped(self) {
-        println!("Actor {} stopped", self.number);
-    }
-}
```

Comment:
This is deleted because the new `Actor` trait no longer allows calling `stop` in `started`.